actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -31,7 +31,7 @@ def new_mail
       end
 
       def mail_params
-        params.require(:mail).permit(:from, :to, :cc, :bcc, :x_original_to, :in_reply_to, :subject, :body, attachments: [])
+        params.expect(mail: [:from, :to, :cc, :bcc, :x_original_to, :in_reply_to, :subject, :body, { attachments: [] }])
```

Comment:
Do we need to explicitly add the hash brackets here?

Reply:
Yes, not valid ruby syntax otherwise.

Reply:
Oh wait, wow, you're right. I didn't realize that _was_ valid.

Reply:
ok, I will undo the changes where I added the hash brackets. Somehow I was so sure this was wrong that I never tried it.

Reply:
Is it? Maybe I'm misunderstanding it, but this works:

```rb
def args(...) = puts(...)

args(a: [:b, c: []])
# {:a=>[:b, {:c=>[]}]}
# => nil
```

Reply:
Fixed and pushed! I'm really happy you called this out. I thought this was ugly but necessary. Now it looks way better and I feel much more happy with the suggested syntax.